From 7aee79f8fca86f45f990b07643a8624562e84fcd Mon Sep 17 00:00:00 2001
From: ozeng <oak.zeng@amd.com>
Date: Thu, 3 Aug 2017 18:41:12 -0400
Subject: [PATCH 5632/5855] drm/amdkfd: Fixed a kfifo adaptation issue

On old linux version there are both kfifo and kfifo-new interfaces.
The adaptation to this was incorrect and caused different
interpretation of struct kfifo in different C source files,
which further caused a kfd interrupt handling issue.

This fixed the issue by adapting kfifo consistently in one
file (kfd_priv.h).

Signed-off-by: Oak Zeng <Oak.Zeng@amd.com>
Reviewed-by: Junwei Zhang <Jerry.Zhang@amd.com>
Reviewed-by: Yong Zhao <Yong.Zhao@amd.com>

Change-Id: I84d32a0a15118eafa158c01d728594987ae0cdd6
---
 drivers/gpu/drm/amd/amdkfd/kfd_interrupt.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_interrupt.c b/drivers/gpu/drm/amd/amdkfd/kfd_interrupt.c
index 258fdda..4f4392a 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_interrupt.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_interrupt.c
@@ -42,11 +42,6 @@
 
 #include <linux/slab.h>
 #include <linux/device.h>
-#if (defined OS_NAME_RHEL) && (OS_VERSION_MAJOR == 6)
-#include <linux/kfifo-new.h>
-#else
-#include <linux/kfifo.h>
-#endif
 #include "kfd_priv.h"
 
 #define KFD_IH_NUM_ENTRIES 8192
-- 
2.7.4

